tests/src/test/scala/cats/tests/SyntaxSuite.scala

```diff
@@ -358,5 +360,22 @@ object SyntaxSuite extends AllSyntaxBinCompat with AllInstances with AllSyntax {
 
     val binested: Binested[F, G, H, A, B] = fgahb.binested
   }
+
+  def testNonEmptySet[A: Order, B: Order] : Unit = {
+    val f = mock[A => B]
+    val set = mock[SortedSet[A]]
+
+    val nes: Option[NonEmptySet[A]] = set.toNes
+    val grouped: SortedMap[B, NonEmptySet[A]] = set.groupByNes(f)
+  }
+
+  def testNonEmptyList[A: Order, B: Order] : Unit = {
```

Comment:
I think we don't need this `Order` constraint on `A`, leaving it on `B` should be enough. :)

Reply:
Thanks, @LukaJCB.
Let me remove it, I will also rebase on top of the latest master before squashing the commits and push.